internal/controller/suite_util_test.go

```diff
@@ -593,11 +602,13 @@ func createDockerConfigSecret(secretKey types.NamespacedName, dockerConfigData s
 
 // generateDockerConfigJson creates the raw JSON string for .dockerconfigjson
 func generateDockerConfigJson(registry, username, password string) string {
-	authString := fmt.Sprintf("%s:%s", username, password)
-	encodedAuth := base64.StdEncoding.EncodeToString([]byte(authString))
-
 	auths := map[string]dockerConfigAuth{}
-	auths[registry] = dockerConfigAuth{encodedAuth}
+
+	if registry != "" && username != "" && password != "" {
```

Comment:
What's the point in calling this method with an empty param?

Reply:
added comment about it